packages/apputils/src/runningSessions.tsx

```diff
@@ -45,10 +45,14 @@ function RunningSessionsComponent(
       onKeyDown={props.handleKeyDown}
       style={{ cursor: 'pointer' }}
     >
-      <GroupItem spacing={HALF_SPACING}>
-        <TextItem source={props.terminals} />
-        <terminalIcon.react verticalAlign="middle" stylesheet="statusBar" />
-      </GroupItem>
+      {props.terminals > 0 ? (
+        <GroupItem spacing={HALF_SPACING}>
+          <TextItem source={props.terminals} />
+          <terminalIcon.react verticalAlign="middle" stylesheet="statusBar" />
+        </GroupItem>
+      ) : (
+        <div></div>
```

Comment:
Can we return `null` or otherwise at least an empty fragment (`<></>`) here? See https://react.dev/learn/conditional-rendering#conditionally-returning-nothing-with-null

```suggestion
        null
```

Reply:
Looks like the use of `GroupItem` prevents this:

```
Type 'Element | null' is not assignable to type 'Element | (Element & string)'.
  Type 'null' is not assignable to type 'Element | (Element & string)'.
```

Reply:
Thanks for checking, I've opened https://github.com/jupyterlab/jupyterlab/issues/17212